PetShare/Models/Announcements/AnnouncementFilters.cs

```diff
@@ -15,6 +18,7 @@ public static AnnouncementFilters FromRequest(GetAllAnnouncementsFilteredQueryRe
     {
         return new AnnouncementFilters
         {
+            Status = Enum.TryParse<AnnouncementStatus>(request.Status, out var outStatus) ? outStatus : null,
```

Comment:
This should accept lowercase statuses (probably). We can use an overload with additional second argument for that.

Also, I guess we should validate this status somewhere and return 400 if it's wrong. We could add `IValidatableObject` to `GetAllAnnouncementsRequest`, or maybe have this method return `null` (or `Result`) if state is invalid and handle it in correctly controller.